monitoring/alerts/spark-rules.yml

```diff
@@ -89,10 +89,10 @@ groups:
       annotations:
         description: 'Spark job's launch rate has unusually high standard deviation'
 
-    - alert: Spark_Job_Success_Rate
-      expr: (sum(spark_mesos_cluster_driver_finished_count_mesos_state_task_finished) / (sum(spark_mesos_cluster_driver_finished_count_mesos_state_task_finished) + sum(spark_mesos_cluster_driver_finished_count_mesos_state_task_lost) + sum(spark_mesos_cluster_driver_finished_count_mesos_state_task_failed))) >= 99.0
+    - alert: Spark_Jobs_Have_Failed
+      expr: (sum(spark_mesos_cluster_driver_finished_count_mesos_state_task_finished) / (sum(spark_mesos_cluster_driver_finished_count_mesos_state_task_finished) + sum(spark_mesos_cluster_driver_finished_count_mesos_state_task_lost) + sum(spark_mesos_cluster_driver_finished_count_mesos_state_task_failed))) < 0.999
```

Comment:
Was the product requirement 99.9%? I thought it was 99%?

Reply:
yeah, I checked back and it was 
> MUST have 99.9% of Spark jobs succeed the first time

I believe this is the most up to date ticket w/ all the PM requirements: https://jira.mesosphere.com/browse/DCOS-28030